assignment-8-flipkart-page-1/index.html

```diff
@@ -0,0 +1,799 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <title>Flipkart</title>
+  <link rel="icon" href="img/icon.png">
+</head>
+
+<body>
+  <header>
+    <div class="header1">
+      <nav class="navigation1">
+        <ul>
+          <li><a href=""><img src="img/h1.PNG"></a></li>
+          <li>
+            <div class="search"><input class="searchbar" type="text" placeholder=" Search for products brands and more">
+              <img src="img/search.svg" width="18">
+            </div>
+          </li>
+          <li><button class="login" type="button">Login</button></li>
+          <li>
+            <div class="dropdown">
+              <button class="dropbtn">More</button>
+              <div class="dropdown-content">
+                <a href="#">Notification Preferences</a>
+                <a href="#">Sell on Flipkart</a>
+                <a href="#">24*7 Customer Care</a>
+                <a href="#">Advertise</a>
+                <a href="#">Download App</a>
+              </div>
+            </div>
+          </li>
+          <li>
+            <div class="carticon">
+              <div>
+                <a class="cart" href="#"><i class="large material-icons">shopping_cart</i></a>
+              </div>
+              <div>
+                <a class="cart" href="#">Cart</a>
+              </div>
+            </div>
+          </li>
+
+        </ul>
+      </nav>
+    </div>
+
+    <div class="header2">
+      <nav class="navigation2">
+        <ul>
+          <li>
+            <div class="d1">
+              <button class="Electronics"><a href="">Electronics</a></button>
+              <div class="electronics_dropdown">
+                <div class="electronics_content one">
+                  <a href="">Mobiles</a>
+                  <a href="">Mi</a>
+                  <a href="">Realme</a>
+                  <a href="">Samsung</a>
+                  <a href="">Infinix</a>
+                  <a href="">OPPO</a>
+                  <a href="">Apple</a>
+                  <a href="">Vivo</a>
+                  <a href="">Honor</a>
+                  <a href="">Asus</a>
+                  <a href="">realme 6</a>
+                  <a href="">Samsung S10 Lite</a>
+                  <a href="">POCO X2</a>
+                  <a href="">Infinix s5 Pro</a>
+                  <a href="">realme X50 Pro</a>
+                  <a href="">realme 6 Pro</a>
+                  <a href="">OPPO F15</a>
+                  <a href="">IQOO 3</a>
+                </div>
+                <div class="electronics_content two">
```

Comment:
try to improve the class names. one, two, three, four etc doen't make sense. try to think in modules and submodule as classes and subclasses.